presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorMetadata.java

```diff
@@ -130,18 +131,26 @@
     private final MetadataDao dao;
     private final ShardManager shardManager;
     private final String connectorId;
+    private final Consumer<Long> deleteCallback;
```

Comment:
maybe name this `beginDeleteCallback` or `beginDeleteListener` or `beginDeleteMonitor`